### PR TITLE
Edgarbranch

### PR DESCRIPTION
**Pull request by Edgarkimbugwe**
Apart from proof reading through, these are the parts that remaining to have the Readme done

- Responsive image
- Edit "Back to top"
- Budget page image
- And images to show testing the page (I will be glad if someone provides me these images and brief explaination on the tests)

### CODE REVIEW

File: README.md
Comment (CamCove-Crichton) [inline README.md:1]: LGTM

File: main/settings.py
Comment (CamCove-Crichton) [inline main/settings.py:1]: This needs to be changed back to 'localhost'. Please could you change it and send the PR again?

File: static/images/readme/expenses-page.png
Comment (CamCove-Crichton) [inline static/images/readme/expenses-page.png:1]: LGTM

**Review by CamCove-Crichton — CHANGES_REQUESTED** [review]
@Edgarkimbugwe could you change the ALLOWED_HOSTS back from your local host url, back to 'localhost' please?

**Review by Edgarkimbugwe — COMMENTED** [review]
I have made more changes and settings in main set back to localhost

**Review by CamCove-Crichton — APPROVED** [review]
LGTM

**Pull request merged**